src/CombinedSubscription.js

```diff
@@ -0,0 +1,74 @@
+import HistoricalSubscription from './HistoricalSubscription'
+import RealTimeSubscription from './RealTimeSubscription'
+import AbstractSubscription from './AbstractSubscription'
+
+export default class CombinedSubscription extends AbstractSubscription {
```

Comment:
Do we need to `extend`? This class seems to be mostly about delegating to `this.sub` and not using many of the facilities provided by the base class?